app/views/devise/registrations/new.html.erb

```diff
@@ -1,57 +1,56 @@
-<div class="container-fluid no-padding">
-  <h1> Add User </h1>
-</div>
+<div class="login">
```

Comment:
This doesn't seem to be the appropriate class name

Reply:
The view files for ['Forgot your password?'](https://github.com/commit-tech/nussu-logistics/blob/e681fc6d9a9092d07fe0480dcdef9fcec8494db0/app/views/devise/passwords/new.html.erb) (app/views/devise/passwords/new.html.erb) and ['unlock instructions'](https://github.com/commit-tech/nussu-logistics/blob/e681fc6d9a9092d07fe0480dcdef9fcec8494db0/app/views/devise/passwords/new.html.erb) (app/views/devise/unlocks/new.html.erb) use the same class name and I adhered to it for consistency. 

The [login class](https://github.com/commit-tech/nussu-logistics/blob/e681fc6d9a9092d07fe0480dcdef9fcec8494db0/app/assets/stylesheets/login.scss) in the css file also has an extensive definition and I am hesitant to create a new file entirely. An idea I have is instead of calling the class 'login', we can call it 'pre_session'

Reply:
Sounds like a good idea. Can you port over all the views with this new naming convention?

Reply:
Ported!